standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/PresentationFramework.xml

```diff
@@ -30924,7 +30924,7 @@ To a <see cref="T:System.Uri" /> that is not located at the site of origin, or
       <summary>The values of this dependency property should be saved or restored by journaling processes, or when navigating by Uniform resource identifiers (URIs).</summary>
     </member>
     <member name="F:System.Windows.FrameworkPropertyMetadataOptions.None">
-      <summary>No options are specified; the dependency property uses the default behavior of the Windows Presentation Foundation (WPF) property system.</summary>
+      <summary>No options are specified; the dependency property uses the default behavior of the WPF property system.</summary>
```

Comment:
This seems incorrect.

The summary contains the whole acronym: https://docs.microsoft.com/en-us/dotnet/api/system.windows.frameworkpropertymetadataoptions?view=netcore-3.0

cc @mairaw

Reply:
This one is weird because the xml contains the same text we see here: https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows/FrameworkPropertyMetadataOptions.xml

Reply:
In other words, the intellisense is matching docs. Not sure why the website is not showing the same. Maybe it autodetects "WPF" and transforms it to the whole acronym? @mairaw

Reply:
Where do you see the intellisense matching the docs?
https://github.com/dotnet/dotnet-api-docs/blob/9ea0edef8fe9034aee6b811dda7f6ee4ee9f55a7/xml/System.Windows/FrameworkPropertyMetadataOptions.xml#L22

> <summary>Specifies the types of framework-level property behavior that pertain to a particular dependency property in the Windows Presentation Foundation (WPF) property system.</summary>